calculators/election_totals.rb

```diff
@@ -45,6 +48,12 @@ def fetch
             by_origin = by_origin.merge(cbo){|key, oldval, newval| oldval + newval}
           end
 
+          # Sum contribution by type information for whole election
+          cbo = candidate.calculation(:contributions_by_type)
```

Comment:
cbo = ?? Looks like the variable is copied from above even though this refers to contributions by type. I find acronyms really hard to deal with in code, it'd be helpful to me to spell these out.